### PR TITLE
Find whether a number is Palindrome or not

### LINKED ISSUE #1718: Pallindromes in Java/Python
Hi! I want to contribute by writing code for this program. It will:
1. Check if a number is palindrome or not
2.Check if a name/word is palindrome or not
Plz assign me this as a participant of GSSOC20.
@SakshiShreya @singh-shreya6 @asha15 @raghav-dalmia

### PR DESCRIPTION
**Pull request by raksha009**
Issue no #1718 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added a new code
- 

#### Languages Used:
- Python
- 

#### Files Added:
- palindrome_no.py
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: 1-D_Array/Python/EvenOdd.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#Python program to count even and odd numbers in an array
```
Comment (jainaman224) [inline 1-D_Array/Python/EvenOdd.py]: Fix file name

File: 1-D_Array/Python/EvenOdd.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+# To count number of even elements
+def  no_of_even_elem(arr):
+
+    #Initialization
```
Comment (jainaman224) [inline 1-D_Array/Python/EvenOdd.py]: Add space after `#`.

File: Palindrome/pallindrome_no.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#Python program to find whether a number is a pallindrome or not
```
Comment (jainaman224) [inline Palindrome/pallindrome_no.py]: Typo in file name.

Comment (raksha009) [reply Palindrome/pallindrome_no.py]: I have fixed the typo, plz review the changes

**Comment by raksha009** [comment]
@asha15  @jainaman224 plz review my PR. I have made the changes. Thanks

**Comment by raksha009** [comment]
@SakshiShreya maam, plz review my PR. Thanks

**Comment by raksha009** [comment]
@singh-shreya6 maam, plz review my PR. Thanks

File: Palindrome/palindrome_no.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Python program to find whether a number is a palindrome or not
+
+# Function to reverse a number
+def reverse(num):
+
+    # Initialize variable
+    rev=0
```
Comment (somya-kapoor) [inline Palindrome/palindrome_no.py]: Add space around =

File: Palindrome/palindrome_no.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+    while ( num > 0 ) :
+        dig = num % 10
+        rev = rev * 10 + dig
+        num = num // 10
+    return rev    # Returning reversed number
+
+    
+# --- main --- 
+num=int(input("Enter a number:"))
```
Comment (somya-kapoor) [inline Palindrome/palindrome_no.py]: Add space around =

File: Palindrome/palindrome_no.py [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+a  = reverse(num)
+
+if(a == num):  #Comparing the reversed number with original number
+    print("Number entered is palindrome!")
+else:
+    print("Number entered is not a palindrome!")
+
+
```
Comment (somya-kapoor) [inline Palindrome/palindrome_no.py]: Remove the line.

File: Palindrome/palindrome_no.py [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+def reverse(num):
+
+    # Initialize variable
+    rev=0
+
+    while ( num > 0 ) :
+        dig = num % 10
+        rev = rev * 10 + dig
+        num = num // 10
+    return rev    # Returning reversed number
```
Comment (somya-kapoor) [inline Palindrome/palindrome_no.py]: move comments above the statement

File: Palindrome/palindrome_no.py [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+    
+# --- main --- 
+num=int(input("Enter a number:"))
+a  = reverse(num)
+
+if(a == num):  #Comparing the reversed number with original number
```
Comment (somya-kapoor) [inline Palindrome/palindrome_no.py]: move comments above the statement

File: Palindrome/palindrome_no.py [hunk 9]
```diff
@@ -0,0 +1,35 @@
+# Python program to find whether a number is a palindrome or not
+
+# Function to reverse a number
+def reverse(num):
+
```
Comment (somya-kapoor) [inline Palindrome/palindrome_no.py]: Remove this line

File: Palindrome/palindrome_no.py [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+    # Initialize variable
+    rev=0
+
+    while ( num > 0 ) :
+        dig = num % 10
+        rev = rev * 10 + dig
+        num = num // 10
+    return rev    # Returning reversed number
+
+
```
Comment (somya-kapoor) [inline Palindrome/palindrome_no.py]: Remove this line

**Comment by raksha009** [comment]
@somya-kapoor maam, done the changes. Plz review

File: Palindrome/palindrome_no.py [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+        rev = rev * 10 + dig
+        num = num // 10
+    # Returning reversed number
+    return rev    
+    
+# --- main --- 
+num = int(input("Enter a number:"))
+a  = reverse(num)
+
+#Comparing the reversed number with original number
```
Comment (somya-kapoor) [inline Palindrome/palindrome_no.py]: Add a space after #

**Comment by somya-kapoor** [comment]
@raksha009 After making the changes, please squash the commits.

File: Palindrome/palindrome_no.py (hunk 11 above)
Comment (raksha009) [reply Palindrome/palindrome_no.py]: done with the changes @somya-kapoor  maam

**Comment by raksha009** [comment]
Maam, i can't sqyash the commits because i have already deleted the branch on my terminal. And the link is broken now

**Comment by raksha009** [comment]
plz merge my PR.
@somya-kapoor @singh-shreya6 @asha15 @vaibhav4595

**Comment by raksha009** [comment]
@raghav-dalmia plz review my PR.

**Pull request merged**